docs/Reference/API-Objects.md

```diff
@@ -189,9 +189,22 @@ Returned by [eth_getTransactionReceipt](API-Methods.md#eth_gettransactionreceipt
 |-------|:-----------------:|---------|
 | **root** | Data, 32&nbsp;bytes| Post-transaction stateroot|
 
+## Transaction Trace Object 
+
+Returned by [`trace_replayBlockTransactions`](API-Methods.md#trace_replayblocktransactions). 
+
+| Key                                  | Type                         | Value                                                                                |
+|-----------------------               |:--------------------:        |--------------------------------------------------------------------------------------|
+| **output**                           | ?                            | Transaction result 
+| **stateDiff**                        | Object                       | [State changes in the requested block](../Concepts/Transactions/Trace-Types.md)
```

Comment:
Minor. You may want to link to the exact heading in the topic.

Reply:
Done